docs/src/blockchain-development/access_control.md

```diff
@@ -0,0 +1,18 @@
+# Access Control
+
+Smart contracts require the ability to restrict access to and identify certain users or contracts.
+
+Unlike account based blockchains, Fuel calls can originate from an origin which has potentially multiple parties.
+
+## msg_sender
+
+To deliver an akin experience to Ethereum based access control, the `std` library provides a `msg_sender` method, which determines a single sender based upon the call and transaction input data.
```

Comment:
"To deliver an  experience akin to Ethereum ..."

Reply:
That sounds better :)